src/locales/en/messages.json

```diff
@@ -103,20 +102,14 @@
     "heroSubtitle": "Learn coding in Arabic, Amharic, Swahili and more.",
     "comingSoon": "Coming Soon",
     "followOnTwitter": "Follow us on Twitter",
-    "getIntouch": "Get in touch to record a tutorial, sponsor the project or just for a chat:",
+    "getIntouch": "Get in touch to record a tutorial, support the project or just for a chat:",
     "subscribe": "Subscribe",
     "notifyMeWhenLaunch": "Notify me when Barmaga is launched",
     "subscribeSuccess": "Your email was added successfully.",
-    "subscribeError": "Something went wrong, please try again or contact us directly through email or Twitter.",
-    "supportUs": "Support Barmaga"
+    "subscribeError": "Something went wrong, please try again or contact us directly through email or Twitter."
   },
   "moreCourses": "See More",
-  "upcomingCourse": {
-    "interestedInCourse": "Interested in this course?",
-    "courseComingSoon": "Course coming soon",
-    "notifyMe": "Notify me of new courses",
-    "feedbackReceived": "👍🏿 Thanks for the feedback."
-  },
-  "yes": "yes",
-  "no": "no"
+  "quote1": "Education is the most powerful weapon which you can use to change the world.",
```

Comment:
no need to add the quotes in the messages file .. just keep it in the footer component, as we will move it next to strapi and retrieve from the backend, with its translations (so that we don't have to change code every time we want to modify the quotes)